cmd/envtool/tests.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"io"
+	"os"
+	"os/exec"
+	"sort"
+	"strings"
+)
+
+// shardIntegrationTests shards integration test names from the specified path.
+func shardIntegrationTests(w io.Writer, index, total uint) error {
+	var output strings.Builder
+
+	testNames, err := shardTests(index, total, "integration")
+	if err != nil {
+		return err
+	}
+	shardedTestNames := testNames[index:total]
```

Comment:
In `shardTests` L51, it takes slice `testNames[index:total]` I think we shouldn't do this here again right?